tests/conftest.py

```diff
@@ -63,22 +64,27 @@ def test_aws_user():
 
 
 @pytest.fixture
-def dynamodb_class():
+def article_instance():
```

Comment:
While I think `article_instance` is _technically_ more accurate, I still might suggest something like `sample_article`.  Or even just `article`, except that's _so_ generic and out of context, I'm not sure what's unique about it.

I'd expect to see `_class` or `_instance` maybe in some complex code, juggling classes and their instances:

Something like:
```python
suite_name = "club"
for suite_class in [Club, Spade, Diamond, Heart]:
    if suite_name == suite_class.name:
        suite = suite_class()
# more work with suite instance here...
``` 

But even here it felt awkward to have `suite_instance` and `suite` felt more ergonomic.

And I hate to say it -- given the numerous changes throughout -- but the same feels true for:
  * `s3_class` --> `s3_instance`
  * `ses_class` --> `ses_instance`
  * etc.

What these files / classes feel like are **clients**, and it's not surprise that `boto3` uses `client` as well, e.g.:

```python
s3_client = boto3.client("s3")
```

When I see `_instance` in these cases, it's a bit confusing because what is an "instance" of S3?  or an "instance" of Dynamo?  But if I saw "client", or even just `s3`, `ses`, etc., I'd have a hunch they were meant to provide methods to interact with those things.

Reply:
You're right, `client` does seem more accurate, updating these!